### PR TITLE
HOTT-1997: Adds opensearch clusters for development, staging and production

### PR DESCRIPTION
**Pull request by willfish**
### Jira link

https://transformuk.atlassian.net/browse/HOTT-1997

### What?

I have added/removed/altered:

- [x] Added development search cluster
- [x] Added staging search cluster
- [x] Added production search cluster

### Why?

I am doing this because:

- This is required to move off of the Paas and get the features we need from opensearch.
- An added benefit is we save lots and lots of money for HMRC in the long run

### CODE REVIEW

File: environments/development/opensearch/terragrunt.hcl [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+inputs = {
+  cluster_name    = "tariff-search-development"
+  cluster_domain  = "trade-tariff.service.gov.uk"
+  cluster_version = "1.3"
+
+  master_instance_enabled = false
+  warm_instance_enabled   = false
+  instance_count          = 3
+  instance_type           = "m6g.xlarge.search"
```
Comment (alexdesi) [inline environments/development/opensearch/terragrunt.hcl:18]: should we use a `xlarge`  for dev?

Comment (willfish) [reply environments/development/opensearch/terragrunt.hcl:18]: This is actually 2/3rds of the costs we're currently paying per instance

Comment (willfish) [reply environments/development/opensearch/terragrunt.hcl:18]: Then you half that because we've merged to using a single cluster per space

**Review by alexdesi — APPROVED** [review]
So, not at all an expert here!
the header says: Adds opensearch clusters for development, staging and production

but I can see stuff only for dev and staging 🤔 

it looks good anyway, approved because I trust you ✅

**Review by alexdesi — APPROVED** [review]
lgtm ✅

**Pull request merged**